Swift/Controllers/XMLConsoleController.cpp

```diff
@@ -41,5 +45,11 @@ void XMLConsoleController::handleDataWritten(const SafeByteArray& data) {
         xmlConsoleWidget->handleDataWritten(data);
     }
 }
-
+void XMLConsoleController::setClient(std::shared_ptr<Client> client_)
+{
+    this->client_ = client_;
+}
+void XMLConsoleController::sendXML(std::string data){
+    client_->sendData(data);
```

Comment:
This should probably check if `client_` is actually set, not? If the developer forgets to call `setClient(…)` pressing the send XML button would cause a crash I think.